cli/pcluster/config_sanity.py

```diff
@@ -372,6 +372,17 @@ def validate(self, resource_type, resource_value):  # noqa: C901 FIXME
                             sys.exit(1)
             except ClientError as e:
                 self.__fail(resource_type, e.response.get("Error").get("Message"))
+        if resource_type == "EC2IAMPolicy":
```

Comment:
Does this new request for retrieving the IAM policy have any impact on the  permissions of the user?